client/src/get-compiled-template.js

```diff
@@ -32,11 +23,20 @@ module.exports = function(cache){
             }
           });
         }
-        var type = template.type;
-        if (type === 'jade') { type = 'oc-template-jade'; }
-        if (type === 'handlebars') { type = 'oc-template-handlebars'; }
 
-        cb(null, templateEngines[type].getCompiledTemplate(templateText, template.key));
+        var type = template.type;
+        var ocTemplate;
+        try {
+          if (type === 'jade') { type = 'oc-template-jade'; }
+          if (type === 'handlebars') { type = 'oc-template-handlebars'; }
+
+          // dynamically require specific oc-template
```

Comment:
I would remove this comment. If this is the only thing in the try, I think it's quite clear what's happening here.